src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/VSCodeAnalyzerLoader.cs

```diff
@@ -30,7 +30,6 @@ public VSCodeAnalyzerLoader(IDiagnosticAnalyzerService analyzerService, IDiagnos
 
     public void InitializeDiagnosticsServices(Workspace workspace)
     {
-        _ = ((IIncrementalAnalyzerProvider)_analyzerService).CreateIncrementalAnalyzer(workspace);
```

Comment:
more hacks from componetns trying to get diagnostics to work, but where they couldn't depend on solution crawler to initialize state.